tests/go/test_appengine_flex_python.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2016 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import unittest
+
+import google.cloud.logging
+
+from ..common.common import Common
+
+
+class TestAppEngineFlex(Common, unittest.TestCase):
+
+    environment = "appengine_flex_python"
+    language = "go"
```

Comment:
Do you need this file? In Python I made two different test environments for the Python runtime and the container runtime. I'm not sure if Go is the same situation. I don't think we need an exact match of all of these test files though

Reply:
yes . will remove thank you